### ISSUE TITLE
Using wifi or ethernet on the CM3+ through the nebra board

### ISSUE THREAD

**Issue by clementpeleman** [NebraLtd/nebra-lora-hardware#9]
Hi all

I have a nebra indoor helium miner with a rpi compute module 3+ as its brains. I would really like to use this module for other projects but I cannot get it to connect to the internet. There is a usb wifi adapter located on the board as well as an ethernet port. I have tried to flash the board with the specified config.txt:

```
core_freq=250
dtoverlay=spi1-3cs
dtoverlay=uart0,txd0_pin=32,rxd0_pin=33,pin_func=7
dtparam=i2c_arm=on
dtparam=spi=on
dtparam=audio=on
avoid_warnings=1
disable_overscan=1
disable_splash=1
gpu_mem=16
```

But it does not seem to use wifi nor am I able to connect it to ethernet. When I plug in a cable no activity lights show up.
The compute module has a status led on the board and blinks irregularly when booting but then goes to blinking once a second roughly forever.
When unplugging the ethernet and trying to use the wifi usb adapter (with wpa_supplicant set) the same applies.

I have also tested the following config:
`dtoverlay=dwc2,dr_mode=host`

I flash the board through the rpi imager software.

Any help is welcome!
Thanks in advance

**Comment by shawaj** [comment]
Did the ethernet and wifi work when it was running on our helium miner software?

Are you sure the daughterboard is fully connected to the motherboard?

And the compute module is seated properly on the daughterboard?

**Comment by clementpeleman** [comment]
Yes, yes and yes... I double checked everything. The blue boot led is blinking irregular, indicating some activity on the board is happening.
